dask_sql/physical/rel/custom/wrappers.py

```diff
@@ -0,0 +1,493 @@
+# Copyright 2017, Dask developers
+# Dask-ML project - https://github.com/dask/dask-ml
+"""Meta-estimators for parallelizing estimators using the scikit-learn API."""
+import logging
+import warnings
+
+import dask.array as da
+import dask.dataframe as dd
+import dask.delayed
+import numpy as np
+import sklearn.base
+import sklearn.metrics
```

Comment:
Should we wrap these in a `try`/`except` similar to what we were doing for dask-ml since scikit-learn isn't a core dependency?